src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,182 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import duke.command.AddDeadlineCommand;
+import duke.command.AddEventCommand;
+import duke.command.AddToDoCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkDoneCommand;
+import duke.command.UnmarkDoneCommand;
+import duke.exception.DukeException;
+
+/**
+ * The Parser class deals with making sense of the user command.
+ *
+ * @author Inez Kok
+ */
+public class Parser {
+    /**
+     * Parses through user command and make sense of it.
+     *
+     * @param input This is the user input.
+     * @param size This is the size of the current task list.
+     * @return Command This returns the respective command based on user input.
+     * @throws DukeException On input error.
+     */
+    public static Command parse(String input, int size) throws DukeException {
+        if (input.equals("bye")) {
```

Comment:
There seems to be a lot of if-else statements here. Would it be better to use a switch statement?